.github/workflows/lecture_participation.yml

```diff
@@ -0,0 +1,36 @@
+name: Lecture Participation Information
+
+on:
+  issue_comment:
+    types: [created]
+
+permissions:
+  issues: write
+
+jobs:
+  track-participation:
+    if: github.event.issue.title == 'Lecture participation tracker 2024'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Addressed in commit ab6b4e5

Reply:
Thanks @bjornthiberg ! You can always use "commit suggestion" in order to approve these changes instead of committing separately.
![image](https://github.com/user-attachments/assets/590a5ff6-c4d7-461e-a7d8-4e0790ae482a)